src/main/resources/jenkins/plugins/http_request/HttpRequestGlobalConfig/config.jelly

```diff
@@ -1,9 +1,25 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:d="jelly:define" xmlns:local="local">
+	<d:taglib uri="local">
+		<d:tag name="blockWrapper">
```

Comment:
Should there be a TODO for when the baseline always includes divbasedformlayouts?